### ISSUE TITLE
Docs: "common" files are unavailable when running a subset of the docs.

### ISSUE THREAD

**Issue by jansiegel** [handsontable/handsontable#9534]
### Description
The question from #9528 made me find a problem that's present in the current version of the docs, but will be even more apparent in the "frameworked" version of the docs.

As I described in #9528, the documentation contains some files that are considered "common" between all the versions of the docs. The contents of the "common" directory currently get copied to each of the versions of the docs.
That means that, if we're doing a full build, they're available at:
- `/docs/[common-dir]/[common-file]`
- `/docs/[version]/[common-dir]/[common-file]`

and after #9441, also at:
- `/docs/[version]/[framework]-data-grid/[common-dir]/[common-file]`

plus, after deploying the first version formally supported by the "frameworked" documentation, also at:
- `/docs/[framework]-data-grid/[common-dir]/[common-file]`

Currently, the common files are always fetched from:
- `/docs/[common-dir]/[common-file]`

This causes a problem when building/running a watch script for only _a subset_ of the documentation, for example, by running:
- `DOCS_VERSION=10.0 npm run docs:start:no-cache`
In this case, only the `/docs/[version]/(...)` directory is built, so the "common" files throw 404s, as they're pulled from a non-existing directory.

I think we should take advantage of the "problem" I described in #9528 and try to always fetch the "common" files from the versioned/frameworked directory, so:
- when running `/docs/`, we'd get the files from `/docs/[common-dir]`
- when running `/docs/[framework]-data-grid/`, we'd get the files from `/docs/[framework]-data-grid/[common-dir]`
- when running `/docs/[version]/[framework]-data-grid/`, we'd get the files from `/docs/[version]/[framework]-data-grid/[common-dir]`
- ... and so on

### Links
- Closely related to #9528 
- This task is a (semi-) blocker for #9440 (it makes the example checker return false positive results)

**Comment by wszymanski** [comment]
> TL;DR this problem causes some common SVG images (e.g. dark theme slider) to appear wrong, and examples to not render if the current version is frameworked.

Not really. Images are versioned using `ImageVersioned` component introduced within: https://github.com/handsontable/handsontable/pull/9482. There are some problems with scripts which have been used as common elements.

**Comment by jansiegel** [comment]
https://github.com/handsontable/handsontable/issues/9534#issuecomment-1148704073 seems to no longer be a problem, so I'm closing this issue.
